org-formation/075-security-hub/security-hub-suppress-lambda.yaml

```diff
@@ -0,0 +1,119 @@
+---
+Transform: AWS::Serverless-2016-10-31
+Resources:
+  AWSLambdaPowertoolsApplication:
+    Type: AWS::Serverless::Application
+    Properties:
+      Location:
+        ApplicationId: arn:aws:serverlessrepo:eu-west-1:057560766410:applications/aws-lambda-powertools-python-layer
+        SemanticVersion: 1.17.0
+    Metadata:
+      aws:cdk:path: sechub-finding-suppression/AWSLambdaPowertoolsApplication
+  BatchupdateLambdaFct:
+    Type: AWS::Lambda::Function
+    Properties:
+      Code:
```

Comment:
I'm a little wary of adding python directly to this project instead of creating a separate lambda repo, but I won't block on that since this function is so small.